bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -127,7 +109,28 @@ public void propertyChange(PropertyChangeEvent evt) {
                 mm.addMarker(false, this, Severity.Warning, msg, v, a);
             }
 
-            // Elevation Constraint
+			// Function to create markers that report solver intervals.
+			final BiFunction<String, String, Option<Marker>> createSolverMarker =
+					(cacheName, prefix) -> {
+						final Map<Obs, Union<Interval>> c = v.getSchedule().getCache(cacheName);
+						final Union<Interval>          ws = c.getOrDefault(a.getObs(), new Union<>());
+						final ImList<Interval>         is = DefaultImList.create(ws.getIntervals()).filter(
+							// Remove intervals that span the whole night.
+							i -> i.getStart() <= v.getSchedule().getStart() && i.getEnd() >= v.getSchedule().getEnd()
```

Comment:
I believe this will do the exact opposite of the comment and remove intervals that don't span the entire night.